lib/translator/common.h

```diff
@@ -23,6 +28,34 @@ void DebugLog(const char* format, ...);
 
 void SetDebugCallback(void (*callback)(const char*));
 
+void* AllocPage(uint8_t count);
+
+void SetAllocPageCallback(void* (*callback)(uint8_t));
+
+struct ScsiContext {
+  scsi_defs::LunAddress lun;
+};
+
+struct NvmeCompletionData {
+  nvme_defs::GenericQueueEntryCpl nvme_cpl[3];
+};
+
+class Translation {
+ public:
+  Translation() = default;
+  StatusCode ScsiToNvme(absl::Span<const uint8_t> scsi_cmd,
+                        const ScsiContext& scsi_context);
+  StatusCode NvmeToScsi(const NvmeCompletionData& cpl_data, void* data_in);
```

Comment:
nit: I feel like uses of these might be more readable (i.e. less dyslexia-inducing) if these were renamed `Start`/`Finish` or `Begin`/`End` or something along those lines.